sdlib/d/gc/page.d

```diff
@@ -726,9 +752,18 @@ private:
 					auto newOccupancy = oldOccupancy & bmp[i];
 
 					occupancyMask |= newOccupancy;
-					count += popCount(oldOccupancy ^ newOccupancy);
+					auto toRemove = oldOccupancy ^ newOccupancy;
+					count += popCount(toRemove);
 
-					e.slabData.rawContent[i] = newOccupancy;
+					scope(success) e.slabData.rawContent[i] = newOccupancy;
+
+					if (!ec.supportsMetadata) {
+						continue;
+					}
+
+					finalizeSlabNimble(
+						toRemove & e.slabMetadataFlags.rawContent[i], sc, e,
+						i * 64);
```

Comment:
No need to multiply by 64.

Reply:
I don't follow. Each nimble is 64 slots, so don't I need to multiply by 64 to get the right index?

Reply:
OK, I get it. No need after I switch the parameter to the nimble index, and then multiply by 64 in the function.